datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml

```diff
@@ -153,6 +153,42 @@
     example: <KEYTAB_FILE_PATH>
     type: string
   description: Set the path to your Kerberos key tab file.
+- name: auth_token
+  value:
+    example:
+      reader:
+        type: <READER_TYPE>
+        <OPTION_1>: <VALUE_1>
+        <OPTION_2>: <VALUE_2>
+      writer:
+        type: <WRITER_TYPE>
+        <OPTION_1>: <VALUE_1>
+        <OPTION_2>: <VALUE_2>
+    type: object
+    properties:
+    - name: reader
+      type: object
+      properties: []
+    - name: writer
+      type: object
+      properties: []
+  description: |
+    This allows for the use of authentication information from dynamic sources.
+    Both a reader and writer must be configured.
+
+    The available readers are:
+
+      - type: file
+        options:
+          path (required): the absolute path to the file with which to read from
+
+    The available writers are:
+
+      - type: header
+        options:
+          name (required): the name of the field e.g.: Authorization
+          value (required): the template value e.g.: Bearer <TOKEN>
```

Comment:
Should we default:
- name to `Authorization`
- value to `Bearer <TOKEN>`

Reply:
I don't think so, no